src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java

```diff
@@ -6,25 +6,26 @@
 import com.softserveinc.ita.kuguk.GoogleHomePage;
 
 import static org.testng.Assert.assertTrue;
-
 import java.util.List;
 
 public class GoogleTest extends TestRunner {
 
 	private List<String> searchResultsLinks;
-	private String searchString = "smartphone";
 
-	@BeforeMethod
-	public void doSearch() {
+	@Test
+	public void testGoogleSearch() {
 		searchResultsLinks = new GoogleHomePage()
 				.open()
-				.searchFor(searchString)
+				.searchFor("funny kitten")
 				.getSearchResultsLinks();
-	}
-
+	assertTrue
+	        (searchResultsLinks.toString().toLowerCase().contains("funny kitten")); }
+	

```

Comment:
do not use custom formatting
assertTrue();

code format before commiting

Reply:
fixed